doc/api/test.md

```diff
@@ -3607,6 +3607,22 @@ added:
 
 The name of the test.
 
+### `context.passed` {boolean}
+
+<!-- YAML
+added: v20.12.0
+-->
+
+Indicates whether the test succeeded, this defaults to `true` before the test is executed, eg. in a `beforeEach` hook.
```

Comment:
e.g. is missing the middle dot

Reply:
Fixed, fyi though 
<img width="255" height="264" alt="image" src="https://github.com/user-attachments/assets/a7041a72-ce74-4d2c-8034-b4b1b5313bd9" />

Reply:
Thanks! https://github.com/nodejs/node/pull/59329